google/resource_compute_backend_service.go

```diff
@@ -351,11 +351,15 @@ func expandBackends(configured []interface{}) ([]*compute.Backend, error) {
 		}
 		if v, ok := data["max_rate"]; ok {
 			b.MaxRate = int64(v.(int))
-			b.ForceSendFields = append(b.ForceSendFields, "MaxRate")
+			if b.MaxRate == 0 {
+				b.NullFields = append(b.NullFields, "MaxRate")
+			}
 		}
 		if v, ok := data["max_rate_per_instance"]; ok {
 			b.MaxRatePerInstance = v.(float64)
-			b.ForceSendFields = append(b.ForceSendFields, "MaxRatePerInstance")
+			if b.MaxConnectionsPerInstance == 0 {
```

Comment:
Shouldn't this be `MaxRatePerInstance`?

Reply:
Ouppsi. Forgot to push my uncommitted changes. Good catch